etsin_finder/frontend/js/stores/view/qvain.js

```diff
@@ -893,6 +950,16 @@ export const RestrictionGrounds = (name, identifier) => ({
   identifier,
 })
 
+export const Infrastructure = (name, url) => ({
+  name,
+  url,
+})
+
+export const Infrastructures = (name, url) => ({
+  name,
+  url,
+})
```

Comment:
Seems that this isn't used for anything.

Reply:
used in history/infrastructure/InfrastructureSelection.jsx:36. However Infrastrctures code is duplicate code with Infrastructure, FieldOfScience and FiedsOfScience. In theory these four could be unify in to a single schema.